### DISCUSSION TITLE
Game Map

### DISCUSSION THREAD

**Discussion by zeltas** [okaybenji/text-engine#19] [Ideas]
Any chance of displaying a map of rooms, people, items as you write on the fly to aid in story, design?

**Comment by okaybenji** [comment]
Sorry for the delayed response (anybody know if I can get Github to email me when someone posts to the Discussions?)

I like the idea. I made an Issue for it here: https://github.com/okaybenji/text-engine/issues/25

**Comment by zeltas** [comment]
Kool.
